I qts copy test (#2001)
DFE-Digital/get-into-teaching-app: app/views/content/international-teacher-training-to-english-standards-introducing-iqts.md

---
title: International qualified teacher status (iQTS)
description: |-
 International qualified teacher status (iQTS) is a new teaching qualification backed by England’s Department for Education (DfE). iQTS is equivalent to qualified teacher status (QTS). You can train where you live and work. 
date: "2021-08-06"
image: "media/images/content/hero-images/0003.jpg"
backlink: "../"
keywords: 
    - international
    - QTS
    - iQTS
    - Qualified teacher status
---

## Introducing iQTS

> International qualified teacher status (iQTS) is a new teaching qualification backed by England’s Department for Education (DfE). iQTS is equivalent to qualified teacher status (QTS). You can train where you live and work. 

Reviewer: Might add a bit more info to the third sentence? Something along the lines of 'You can train towards your iQTS from where you live and work.' Might just make it read a bit better? And maybe make the second sentence just 'It is equivalent to qualified teacher status' to avoid too much repetition of iQTS.

## Key facts

1. Apply now for courses starting September 2022

> 2. Train for a year via remote learning and at a school in your region 

Reviewer: I might split point 2 into two separate points just for clarity.

2. Training for your iQTS takes a year
3. The course includes remote and school-based learning

3. Fees for iQTS vary – check with participating iQTS training providers
4. Gain a professional teaching qualification recognised as equivalent to QTS

## Six reasons to apply for iQTS

1. iQTS leads to the award of English qualified teacher status ([QTS](https://www.gov.uk/guidance/qualified-teacher-status-qts)) with no further fees or training, and qualifies you to teach in all schools in England

2. iQTS is open to international candidates from anywhere in the world who meet the eligibility criteria

3. iQTS training and assessment are completed in your home country or school workplace, with no need to visit the UK

> 4. iQTS offers high-quality, evidence-based training delivered by world class English teacher training providers, with in-school support from a trained mentor

Reviewer: Sentence feels a little wordy. Maybe split it up or remove some information? Could be just:
'iQTS offers high-quality, evidence-based training with in-school support from a trained mentor.'

5. iQTS is equivalent to and meets the same teaching standards as QTS – regarded as the teacher training gold standard by British international schools and other schools and employers around the world

6. Unlike an iPGCE, iQTS is a UK-government backed professional qualification which qualifies you to teach in all schools in England

## Learn who iQTS is for

> iQTS is a teaching qualification designed for UK and non-UK citizens living outside the UK. If you are a UK or non-UK citizen living in England, you should [apply for teacher training in England](/steps-to-become-a-teacher). 

Reviewer: ```suggestion
iQTS is a teaching qualification designed for citizens of any country living outside the UK. If you're living in England, you should [apply for teacher training in England](/steps-to-become-a-teacher). 
```
'UK' just seemed a little repetitive so tried to reword it a little

* you have some experience teaching, and would benefit from training and mentoring leading to a professional teaching qualification equivalent to QTS

* you wish to teach in England, and you have a qualification (for example, a PGCE, PGDE or iPGCE) that does not allow you to teach in all schools in England

* you are already a teacher, and you wish to develop professionally and enhance your global employability with an internationally recognised teaching qualification

* you are already a teacher, and would benefit from a professional teaching qualification allowing you to teach in all schools in England

## Apply for iQTS, step by step

### 1. Check your eligibility

To apply for iQTS you will need to have: 

* a bachelor’s degree from a UK or non-UK university

* a maths qualification equivalent to an English GCSE grade 4 

* for teaching children aged 3 to 11, a biology, chemistry or physics qualification equivalent to an English GCSE grade 4 

* excellent written and verbal English language skills

#### Help with qualifications
A GCSE (General Certificate of Secondary Education) is an academic qualification awarded for exams in England, usually taken at age 16. 

> If you’re not sure of the status of your degree in England, or whether your qualifications in maths, biology, chemistry or physics are equivalent to an English GCSE grade 4, visit [UK ENIC](https://www.ecctis.com/Default.aspx), the UK National Information Centre for global qualifications and skills.

Reviewer: ```suggestion
If you’re not sure of the status of your degree in England, or whether your qualifications are equivalent to an English GCSE grade 4, visit [UK ENIC](https://www.ecctis.com/Default.aspx). This is the UK National Information Centre for global qualifications and skills.
```
Removing the specific qualifications makes it a little more succinct and less wordy, feel the sentence still works without the extra info (have already said above specifically which subject qualifications they need)

You can complete your induction in a relevant school in England (see [Induction for early career teachers (England), para 2.1](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/972316/Statutory_Induction_Guidance_2021_final__002_____1___1_.pdf)) or a [DfE-accredited British School Overseas](https://www.gov.uk/government/publications/apply-for-qualified-teacher-status-qts-if-you-teach-outside-the-uk/routes-to-qualified-teacher-status-qts-for-teachers-and-those-with-teaching-experience-outside-the-uk#stat).

## Further information
Learn more about [iQTS policy](https://www.gov.uk/government/publications/international-qualified-teacher-status-iqts/introducing-the-international-qualified-teacher-status-iqts-pilot) and [Teachers’ Standards](https://www.gov.uk/government/publications/international-qualified-teacher-status-teachers-standards). 

> Learn more about [other routes to qualified teacher status (QTS)and teaching in England](https://www.gov.uk/government/publications/apply-for-qualified-teacher-status-qts-if-you-teach-outside-the-uk#apply-to-the-teaching-regulation-agency-tra).

Reviewer: ```suggestion
Learn more about [other routes to qualified teacher status (QTS) and teaching in England](https://www.gov.uk/government/publications/apply-for-qualified-teacher-status-qts-if-you-teach-outside-the-uk#apply-to-the-teaching-regulation-agency-tra).
```
